tests/61push/02add_rules.pl

```diff
@@ -317,3 +317,34 @@ sub check_add_push_rule
          Future->done(1);
       });
    };
+
+
+test "Adding the same push rule twice is idempotent",
+   requires => [ local_user_fixture() ],
+
+   bug => "SYN-391",
+
+   do => sub {
+      my ( $user ) = @_;
+
+      matrix_add_push_rule( $user, "global", "sender", '@bob:example.com', {
+         "actions" => [ "notify" ]
```

Comment:
The quote marks here are unnecessary; as it's a bareword on the LHS of the fat arrow, it'll quote nicely anyway